c2corg_ui/templates/base.html

```diff
@@ -109,6 +118,15 @@
     <script src="${request.static_url('%s/angular-messages/angular-messages.min.js' % node_modules_path)}"></script>
     <script src="${request.static_url('%s/angular-cookies/angular-cookies.min.js' % node_modules_path)}"></script>
     <script src="${request.static_url('%s/typeahead.js/dist/typeahead.bundle.min.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/min/moment.min.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/angular-moment/angular-moment.min.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/ca.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/fr.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/en-gb.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/de.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/es.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/eu.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/it.js' % node_modules_path)}"></script>
     <script src="${request.static_url('c2corg_ui:static/build/build.js')}"></script>
```

Comment:
If the files are the same in both standard and debug modes, you could include them only once, outside of the ``if debug:`` blocks.

Isn't it possible to load those locale files only when the interface lang changes? (as done for angular-gettext).
If not we will have to make sure that those locale files are included in a global libaries build. See https://github.com/c2corg/v6_ui/issues/85